examples/exporting/.gitignore

```diff
@@ -0,0 +1,37 @@
+# Logs
```

Comment:
do we really need multiple copies of this file at this level? wouldn't it be saner to move these rules to the top level .gitignore? Also: do all of these still apply?

Reply:
all the examples are based on the default `skpm` templates, I wanted to get them as close as possible to what they would when using `skpm`